### PR TITLE
Getting the ball rolling with this whole issue template thing.

### PR DESCRIPTION
**Pull request by jmarolf**
We can now do [issue templates](https://help.github.com/articles/creating-an-issue-template-for-your-repository/), should we?

### CODE REVIEW

**Comment by jmarolf** [comment]
@dotnet/roslyn-ide @dotnet/roslyn-compiler @dotnet/roslyn-analysis

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+**Version Used**: Visual Studio 2015 Update 1
+
+**Steps to Reproduce**
+1.
+2.
+3.
+
+**Expected Behavior**
```
Comment (jasonmalinowski) [inline .github/ISSUE_TEMPLATE.md]: Suffix with : since people will probably want to type that

**Comment by lgolding** [comment]
:+1:

**Comment by lgolding** [comment]
Oops! Sorry.

**Comment by TyOverby** [comment]
:-1: for the default "Visual Studio 2015 Update 1"  

Make them double-check their version.

**Comment by davkean** [comment]
@dotnet-bot retest prtest/mac/dbg/unit32 please
// Previous failure: http://dotnet-ci.cloudapp.net/job/roslyn_prtest_mac_dbg_unit32/3896/
// Retest reason: Mono has a bug that occasionally sits in an infinite loop. Retry the test.

**Comment by jmarolf** [comment]
@dotnet-bot retest prtest/win/dbg/eta please
// Previous failure: http://dotnet-ci.cloudapp.net/job/dotnet_roslyn_prtest_win_dbg_eta/4329/
// Retest reason: SplitFile flakiness unrelated

**Comment by dpoeschl** [comment]
@TyOverby Are you okay with the updated version?

**Comment by dpoeschl** [comment]
:+1:

**Comment by jmarolf** [comment]
@dotnet-bot retest prtest/win/dbg/eta please
// Previous failure: http://dotnet-ci.cloudapp.net/job/dotnet_roslyn_prtest_win_dbg_eta/4359/
// Retest reason:

**Pull request merged**